Journal/Journal/ViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ViewController.swift
+//  Journal
+//
+//  Created by Jonathan Ferrer on 6/3/19.
+//  Copyright © 2019 Jonathan Ferrer. All rights reserved.
+//
+
```

Comment:
You don't need this file, its best to delete it so it doesn't occupy space in your project.